packages/iTwinUI-react/src/core/Tile/Tile.tsx

```diff
@@ -73,6 +73,10 @@ export type TileProps = {
    * Dropdown menu containing `MenuItem`s.
    */
   moreOptions?: React.ReactNode[];
+  /**
+   * Status of the tile.
+   */
+  status?: 'positive' | 'warning' | 'negative';
   /**
```

Comment:
Just curious, does the order of defining props matter? Because status is on L79 but loading is on L107.

Reply:
Order should not matter. I just added new props to these lines based on relevance. To me it makes sense to have `status` near other props, like options, thumbnail, etc., and to have `isLoading` near `isActionable`.